### PR TITLE
Excel example

### PR DESCRIPTION
**Pull request by stevenkj1**

### CODE REVIEW

File: ExcelExample-FrenchTraffic/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# FROM continuumio/anaconda3
+# Ok, image will be big, but should work ? Too big ..
+FROM python:3.9
+# ENV ISDOCK="Yes"
```
Comment (roseDickinson) [inline ExcelExample-FrenchTraffic/Dockerfile]: This line and the first and second should probably be deleted

File: ExcelExample-FrenchTraffic/main.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+gDEBUG = False
+gGRAPH = False
+gSAVE = True
+
+gPATHI = ""
+gPATHO = ""
+
+isDAFNI = os.environ.get("ISDAFNI")
+print("ISDAFNI Environment variable = ", isDAFNI, type(isDAFNI))
+if os.environ.get("ISDAFNI") == "True":
```
Comment (roseDickinson) [inline ExcelExample-FrenchTraffic/main.py]: why not use the isDAFNI variable here?

also you can shorten this by using `os.getenv("ISDAFNI")` and optionally provide a second arg to be the default value e.g.: `os.getenv("ISDAFNI", False)`

File: ExcelExample-FrenchTraffic/main.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+    pdat = PlotData(df)
+    if gDEBUG:
+        print(pdat.dataset)
+    if gGRAPH:
+        # Quick pyplot test
+        for yr in range(1970, 1974):
+            plt.errorbar(pdat.rmonths, pdat.mvals_by_year(yr), yerr=pdat.emvals_by_year(yr), label=yr)
+        plt.legend(loc="upper left")
+        plt.show()
+        #for mnth in range(0, len(pdat.rmonths)):
```
Comment (roseDickinson) [inline ExcelExample-FrenchTraffic/main.py]: commented code to remove?

File: ExcelExample-FrenchTraffic/main.py [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+            tva = "Run directory"
+        else:
+            tva = gPATHO
+        print("Saving output data to : ", tva)
+        outdaty = os.path.join(os.path.join(gPATHO, "daty.json"))
+        outdatm = os.path.join(os.path.join(gPATHO, "datm.json"))
+        jfiley = open(outdaty, "w")
+        flin1 = {}
+        flin2 = {}
+        for yr in pdat.ryears: # Mv this to the class
```
Comment (roseDickinson) [inline ExcelExample-FrenchTraffic/main.py]: Should the comment be removed?

There appear to be a lot of comments that could be removed

File: ExcelExample-FrenchTraffic/Dockerfile [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+COPY requirements.txt .
+RUN pip install --no-cache-dir --compile -r requirements.txt
+
+# Create the data path. Rem. upload input data via web interface in DAFNI.
+RUN mkdir -p /data/inputs/
+RUN mkdir -p /data/outputs/
+
+# Copy over code & run. This one needs excel.
+COPY main.py .
+# COPY BAAC_2021.xlsx . # /data/inputs
```
Comment (roseDickinson) [inline ExcelExample-FrenchTraffic/Dockerfile]: This line might need deleting too?

**Review by roseDickinson — CHANGES_REQUESTED** [review]

File: ExcelExample-FrenchTraffic/Dockerfile (hunk 1 above)
Comment (stevenkj1) [reply ExcelExample-FrenchTraffic/Dockerfile]: Removed all the excess commented out lines

**Review by roseDickinson — APPROVED** [review]

**Pull request merged**